app/routes/address_blueprint.py

```diff
@@ -1,7 +1,7 @@
 from flask import Blueprint
-from app.controllers.address_controller import address
 
+from app.controllers.address_controller import address
 
-bp_address = Blueprint('bp_address',__name__,url_prefix='/address')
+bp_address = Blueprint("bp_address", __name__, url_prefix="/address")
 
-bp_address.get('/')(address)
+bp_address.get("/")(address)
```

Comment:
I think we should get rid of this `"/"` and change to only `""`, because that way the request should be made using `/address/`

Reply:
Done.